CHANGELOG-6.md

```diff
@@ -5,27 +5,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic
 Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [6.8.2] - 2022-10-05
 
 ### Changed
 - Agent logs now include check name when a check request is not executed.
+- The Sensu Agent service now auto-restarts after failures on Windows.
+
 ### Added
 - The authentication module now logs successful (INFO) and unsuccessful (ERROR)
   login attempts.
 - Agent websocket connection logging includes backend entity name.
-
-### Changed
-- The Sensu Agent service now auto-restarts after failures on Windows.
+- A sensu.io/output_truncated_bytes label is now applied to events when
```

Comment:
Don't you need to escape the `_` here too then?

Reply:
Ah didn't notice that one, thanks for noticing @ccressent. Updated.